app/backend/aquifers/management/commands/import_licences.py

```diff
@@ -89,20 +89,24 @@ def process_row(self, row, use_dev_fixtures=False, well=None, aquifer=None):
             # rows with POD. POD refers to surface water which is out of scope for GWELLS)
             return
 
-        if not row['SOURCE_NAME'].strip().isdigit():
-            # Licence must be for an aquifer
-            return
-
-        if not row['WELL_TAG_NUMBER'].strip().isdigit():
-            # Licence must be for a well
+        if not row['SOURCE_NAME'].strip().isdigit() and not row['WELL_TAG_NUMBER'].strip().isdigit():
+            # Licence must be for a well or aquifer
             return
 
         logging.info("importing licence #{}".format(row['LICENCE_NUMBER']))
 
         if not well or not aquifer:
             # Check the Licence is for a valid Aquifer and Well
-            aquifer = Aquifer.objects.get(pk=row['SOURCE_NAME'])
-            well = Well.objects.get(pk=row['WELL_TAG_NUMBER'])
+            # the if check here allows this function to be called with a specific
+            # well or aquifer for dev/test environments.
```

Comment:
The comment pertains to the "if" condition, which I'm guessing is above? Could we move that above the `if` statement to clean it up and provide less confusion?